docs/DeveloperGuide.md

```diff
@@ -658,9 +658,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+| **Command**            | A command is an instruction given by a user to LinkMeIn to perform a specific action. For example,`add` command is a command to add the company's application into LinkMeIn.                                                        |
+| **COMPANY_NAME**       | Name of the company that you are applying to.  Only contain alphanumeric characters and spaces, and should not be blank. Maximum of 100 characters (excluding spaces).                                                              |
+| **DEADLINE**           | Deadline of the application. Should be in DD-MM-YYYY format. Dates before the current date are allowed.                                                                                                                             |
+| **EMAIL**              | Email of the recruiter. Should be in the format of `local-part@domain` and should not be blank.                                                                                                                                     |
+| **GUI**                | Graphical User Interface (GUI) is a visual method to interact with software using icons, buttons, and windows. GUI provides a user-friendly way to interact with software using graphical elements rather than text-based commands. |
+| **Index**              | Refers to the index number shown in the displayed company list                                                                                                                                                                      |
+| **JAR**                | JAR stands for Java Archive and is a package file format typically used to aggregate many Java class files and associated metadata and resources into one file for distribution.                                                    |
+| **JSON**               | JSON stands for JavaScript Object Notation. It is lightweight format for data interchange, easy to read and write for humans, and easy to parse for machines. Often used in web applications and configuration files.               |
+| **Mainstream OS**      | Windows, Linux, Unix, OS-X                                                                                                                                                                                                          |
+| **Parameter**          | Parameter is similar to a field in a form you have to fill up. For example, in the command `edit 1 c/COMPANY_NAME e/EMAIL`, `COMPANY_NAME` and `EMAIL` are parameters in the command.                                               |
+| **PHONE_NUMBER**       | Phone number of the recruiter. Only contain numbers, be at least 3 digits and at most 20 digits long. Should not be blank.                                                                                                          |
```

Comment:
Will it be better if it's

_Should only contain numbers, and be...._

Same for lines 679, 681

Reply:
Yes